### DISCUSSION TITLE
I have set up the bank account, clicked the contribute button, this page appeared, how to fix it?

### DISCUSSION THREAD

**Discussion by kimmy-wang** [opencollective/opencollective#5734] [General]
I have set up the bank account, clicked the contribute button, this page appeared, how to fix it?

![截屏2022-07-11 10 39 50](https://user-images.githubusercontent.com/19725091/178178652-5d0efc47-4cd4-4ab4-b4ec-03c0d5959ea3.png)

**Comment by SudharakaP** [comment]
@see : Did you setup the bank account by clicking on the `Start accepting contributions` button? Can you try refreshing the cache of the page. In Chrome and Firefox you can do this by using the key combination `Ctrl + Shift + R`

**Reply by kimmy-wang** [reply]
I have set up a bank account

<img width="914" alt="截屏2022-07-15 20 54 04" src="https://user-images.githubusercontent.com/19725091/179227162-42461775-a0b5-41a2-a7fc-a162be663764.png">

**Reply by SudharakaP** [reply]
Could you please provide the collective name? I'll take a look. 🤔

**Reply by kimmy-wang** [reply]
https://opencollective.com/bootstrap-vue-plus

**Reply by kimmy-wang** [reply]
https://opencollective.com/bootstrap-vue-plus

**Reply by SudharakaP** [reply]
@see : I tried contributing and for me the contribution flow works. Here's a screen-cast; 

https://user-images.githubusercontent.com/12435965/179630043-2d914855-9ae4-44b9-86be-c951fd28d4e0.mp4

My thinking is that previously you encountered a browser cache issue. Can you try again and let us know if you can see this issue? 🤔

**Reply by kimmy-wang** [reply]
You can try to click the first

**Reply by Betree** [reply]
@see Recurring bank transfers are not supported yet (see https://github.com/opencollective/opencollective/issues/2734). These two first tiers (Backer/Sponsor) are recurring-only, therefore there is no payment provider available for them. I'd recommend connecting Stripe, PayPal, or removing the ability to contribute recurring from Admin > Tiers.
